code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -33,126 +37,174 @@ LINEN BINS
+	dream_messages = list("authority", "a silvery ID", "a bomb", "a mech", "a facehugger", "maniacal laughter", "the research director")
+	nightmare_messages = list("toxins full of plasma", "UPGRADE THE SLEEPERS", "rogue ai")
 
 /obj/item/weapon/bedsheet/medical
 	name = "medical blanket"
 	desc = "It's a sterilized* blanket commonly used in the Medbay.  *Sterilization is voided if a virologist is present onboard the station."
 	icon_state = "sheetmedical"
 	item_color = "medical"
+	dream_messages = list("healing", "life", "surgery", "a doctor")
+	nightmare_messages = list("death", "no cryox", "cryo is off")
 
 /obj/item/weapon/bedsheet/cmo
 	name = "chief medical officer's bedsheet"
 	desc = "It's a sterilized blanket that has a cross emblem.  There's some cat fur on it, likely from Runtime."
 	icon_state = "sheetcmo"
 	item_color = "cmo"
+	dream_messages = list("authority", "a silvery ID", "healing", "life", "surgery", "a cat", "the chief medical officer")
+	nightmare_messages = list("chemists making meth", "cryo if off", "where is the defib")
 
 /obj/item/weapon/bedsheet/hos
 	name = "head of security's bedsheet"
 	desc = "It is decorated with a shield emblem.  While crime doesn't sleep, you do, but you are still THE LAW!"
 	icon_state = "sheethos"
 	item_color = "hosred"
+	dream_messages = list("authority", "a silvery ID", "handcuffs", "a baton", "a flashbang", "sunglasses", "the head of security")
+	nightmare_messages = list("the clown", "a toolbox", "sHiTcUrItY", "why did you put them in for 50 minutes")
+
 
 /obj/item/weapon/bedsheet/hop
 	name = "head of personnel's bedsheet"
 	desc = "It is decorated with a key emblem.  For those rare moments when you can rest and cuddle with Ian without someone screaming for you over the radio."
 	icon_state = "sheethop"
 	item_color = "hop"
+	dream_messages = list("authority", "a silvery ID", "obligation", "a computer", "an ID", "a corgi", "the head of personnel")
+	nightmare_messages = list("improper paperwork", "all access clown", "50 open clown slots", "dead ian")
 
 /obj/item/weapon/bedsheet/ce
 	name = "chief engineer's bedsheet"
 	desc = "It is decorated with a wrench emblem.  It's highly reflective and stain resistant, so you don't need to worry about ruining it with oil."
 	icon_state = "sheetce"
 	item_color = "chief"
+	dream_messages = list("authority", "a silvery ID", "the engine", "power tools", "an APC", "a parrot", "the chief engineer")
+	nightmare_messages = list("forced airlock", "syndicate bomb", "explosion in research chem", "iT's LoOsE")
 
 /obj/item/weapon/bedsheet/qm
 	name = "quartermaster's bedsheet"
 	desc = "It is decorated with a crate emblem in silver lining.  It's rather tough, and just the thing to lie on after a hard day of pushing paper."
 	icon_state = "sheetqm"
 	item_color = "qm"
+	dream_messages = list("a grey ID", "a shuttle", "a crate", "a sloth", "the quartermaster")
+	nightmare_messages = list("a blad person", "no points", "wheres the ore", "space carp")

```

Comment:
Should be `a bald person`.

`where's the ore` won't stype properly without the question mark. I'd remove it. No apostrophe otherwise.

Reply:
Orm not upgraded would work better

Reply:
"All miners dead", "Clown weapon crate",

Reply:
theres not apostrophe on it anyway flattist